### ISSUE TITLE
Bug: Can't use play/pause, previous or next in the Groove Music media flyout

### ISSUE THREAD

**Issue by arnavbhate** [ModernFlyouts-Community/ModernFlyouts#695]
**Describe the bug:**

When playing something in Groove Music, the play/pause, previous and next buttons don't work.

**To Reproduce:**

Steps to reproduce the behaviour:
1. Open Groove Music.
2. Play a song.
3. Change volume to trigger the volume and media flyouts.
4. Try clicking on any of the play/pause, previous or next buttons.
5. See error.

**Screenshots:**

![Screenshot (312)](https://user-images.githubusercontent.com/86760472/125165934-c1ffdb00-e1b6-11eb-919f-86bec9ed970f.png)

**OS Version:**
 - Windows 10 Version: Windows 10 21H1 19043.1083

**ModernFlyouts Version:**
 - Version: v0.9.3

**Additional context:**

**Comment by arnavbhate** [comment]
It seems that restarting explorer.exe has the same effect as restarting ModernFlyouts.

**Comment by MajorCarrot** [comment]
I am on 19043.1110 and don't seem to have this problem. Weird

**Comment by arnavbhate** [comment]
It only happens when I playing something, then I "clear now playing", then close Groove Music, reopen it and play something.

**Comment by MajorCarrot** [comment]
Yup, I can confirm having that issue as well. I can also confirm that the default windows flyouts don't have this issue
